racunaljka.js

```diff
@@ -0,0 +1,230 @@
+/*=================================== brojevi==================================================*/
```

Comment:
I can't find "AutoFormatter" but I found "Prettify" and used that. Is it ok?

I set up my editor to use space for indentation but I don't see any difference. Can you see the difference?
Maybe I have to write in my code again?